app/views/qa/show.html.erb

```diff
@@ -28,6 +28,13 @@
             items: nested_options(filter_group)
           } %>
         <% end %>
+      <% elsif multiple_question? %>
+        <%= render "govuk_publishing_components/components/checkboxes", {
+          name: "#{current_facet['facet']['key']}[]",
+          heading: "Select all that apply",
+          visually_hide_heading: true,
```

Comment:
This looks as though it's going to render something like:

```html
<legend class="visually-hidden">Select all that apply</legend>
<span>Select all that apply</span>
```

...which would mean reading out "Select all that apply, Select all that apply" on screen readers.

I think what we actually want is for the `legend` to be "Does your business do any of the following activities?".

If you set `heading: "Does your business do any of the following activities?"` for the checkboxes, we'd end up repeating the page title. But it looks as though you can use that as the `h1` by setting `is_page_heading: true`.

https://govuk-publishing-components.herokuapp.com/component-guide/checkboxes/with_legend_as_page_heading

Reply:
This is the case on the live site too, e.g. https://www.gov.uk/prepare-business-uk-leaving-eu?page=2.  It should probably be fixed as a separate PR.

![Screen Shot 2019-06-25 at 09 04 19](https://user-images.githubusercontent.com/6329861/60080665-45f9de00-9728-11e9-9cc6-b5c0b456d7a8.png)

Reply:
Have raised https://trello.com/c/V5zTnqDY/181-fix-qa-screen-reader-accessibility.